eng/Versions.props

```diff
@@ -263,6 +263,9 @@
     <XUnitRunnerVisualStudioVersion>2.4.3</XUnitRunnerVisualStudioVersion>
     <MicrosoftDataSqlClientVersion>1.0.19249.1</MicrosoftDataSqlClientVersion>
     <MicrosoftAspNetCoreAppVersion>6.0.0-preview.3.21167.1</MicrosoftAspNetCoreAppVersion>
+    <!-- Source build dependencies -->
+    <NuGetVersioningPackageVersion>5.9.0</NuGetVersioningPackageVersion>
+    <MicrosoftBuildFrameworkPackageVersion>16.9.0</MicrosoftBuildFrameworkPackageVersion>
```

Comment:
Revert. We eradicated all `$(WhateverPackageVersion)` property names a while ago.

We already have `$(NuGetVersioningVersion)` and `$(MicrosoftBuildFrameworkVersion)` properties. The first is the slightly newer `5.10.0-rc.7240` version; the second is identical. If we need an older NuGet.Versioning package for some reason, pick a more unique / less confusing name for an added property.

Reply:
I specifically kept the names used in the source build patches since I'm not sure if we needed to match their naming schemes. @MichaelSimons can source build be updated to use the properties recommended by @dougbu ?

Reply:
@JunTaoLuo, Source build handles either suffix today - `$(WhateverPackageVersion)` and `$(WhateverVersion)`.  During the source-build build, the Versions.props file that gets used has entries with both package name formats in order to support both naming conventions.

Reply:
Names in eng/Versions.props match those from Arcade and we shouldn't need multiple versions here.

Reply:
To clarify my earlier comment, I was referring to the entire .NET product source-build infrastructure.  At the repo level, you only need one version entry.  Since ASP.NET uses the `$(WhateverVersion)` convention, then please stick with that and don't add these new `$(WhateverPackageVersion)` entries.

Reply:
Sounds good, I'll make the changes.